Model/QueuedTask.php

```diff
@@ -246,4 +251,32 @@ public function cleanFailedJobs($capabilities) {
 		return $this->deleteAll($conditions, false);
 	}
 
+/**
+ *
+ * @param array $conditions Conditions
+ * @param string $key Key
+ * @param array $values Values
+ * @return array
+ */
+	protected function _addFilter(array $conditions, $key, array $values): array {
+		$include = [];
+		$exclude = [];
+		foreach ($values as $value) {
+			if (substr($value, 0, 1) === '-') {
+				$exclude[] = substr($value, 1);
+			} else {
+				$include[] = $value;
+			}
+		}
+
+		if ($include) {
+			$conditions[$key . ' IN'] = $include;
+		}
+		if ($exclude) {
+			$conditions[$key . ' NOT IN'] = $exclude;
```

Comment:
I don't think the `IN` and `NOT IN` are needed

Reply:
I think this may have been done on purpose. If $conditions['NOT'] is added here, there's a greater chance it would be overwritten by the calling function. Not too clean, though